SixTrack/sixtrack.s

```diff
@@ -13668,7 +13696,7 @@ cc2008
       call enable_xp()
 +ei
       read(ch1,*,round='nearest')                                       &
-     & idat,kz(i),ed(i),ek(i),el(i),bbbx(i),bbby(i),bbbs(i)
+     & idat,kz(i),ed(i),ek(i),el(i),bbbx(i),bbby(i),bbbs(i) !read fort.2 (or fort.3), idat -> bez = single element name, kz = type of element, ed,ek,el = strength, random error on strenght,length (can be anything),bbbx,bbby,bbbs = beam-beam
```

Comment:
Could you put a note that the beam-beam stuff in fort.2 about to die? See: https://github.com/kyrsjo/SixTrack/commit/61d40506a12c83095e58696ead9009fbdf5fbb6c

Just waiting for the BB block output from MadX...